src/super_gradients/common/factories/base_factory.py

```diff
@@ -16,13 +18,14 @@ def get(self, conf: Union[str, dict, list]):
 
                 If provided value is not one of the three above, the value will be returned as is
         """
-        raise NotImplementedError
+        pass
```

Comment:
Why did you remove the Error? This will lead to unexpected behaviour when users use the factory and will get a none value back, resulting in some other error at some point...

Reply:
It is not removed, it is automatically applied by the @abstractmethod decorator and the @ABC inheritance
This is a better practice for abstract classes and methods, and gives additional useful properties when working with them (e.g. instructions by IDE, auto-complete, etc.)